specs/market-api.yaml

```diff
@@ -641,44 +727,93 @@ paths:
           to return to the Negotiation phase by sending a new Proposal.
 
         * `Cancelled` - Indicates that the Requestor himself has called
-         `cancelAgreement`, which effectively stops the Agreement handshake.
-
+          `cancelAgreement`, which effectively stops the Agreement handshake.
       responses:
-        '200':
+        200:
           description: Agreement approval result.
           content:
             application/json:
               schema:
                 type: string
                 enum: [Approved, Rejected, Cancelled]
```

Comment:
@nieznanysprawiciel the solution for your concern is to return here a single event, but then we will not be back compatible, so it is not possible in this PR which is meant for `Alpha.3` 
```
                $ref: '#/components/schemas/AgreementEvent'
```

Reply:
+ in ApproveAgreement on Provider

Reply:
postponed to #53